src/Credentials.php

```diff
@@ -89,6 +94,12 @@ private static function validateCredentials($credentials) {
             return $credentials;
         }
 
+        private function maybeInitializeAccessToken(){
```

Comment:
Can you change the name of this?  `initializeAccessToken` would be fine.

Reply:
I have updated the method name as requested.